### PR TITLE
chore: update dm ci

### PR DESCRIPTION
**Pull request by Ehco1996**
* split other test
* support setup mysql ssl certs

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]

File: jenkins/pipelines/ci/dm/dm_ghpr_new_test.groovy [hunk 1]
```diff
@@ -212,10 +316,9 @@ def run_single_it_test(String case_name) {
                                     envVar(key: 'MYSQL_ROOT_PASSWORD', value: "${MYSQL_PSWD}"),
                             ],
                             args: "${MYSQL_ARGS}"),
-                    // mysql 8.0
```
Comment (lance6716) [inline jenkins/pipelines/ci/dm/dm_ghpr_new_test.groovy:215]: did you consult someone before you make this change?

**Review by lance6716 — CHANGES_REQUESTED** [review]

File: jenkins/pipelines/ci/dm/dm_ghpr_new_test.groovy (hunk 1 above)
Comment (Ehco1996) [reply jenkins/pipelines/ci/dm/dm_ghpr_new_test.groovy:215]: this pr is WIP, and I was going to ask someone who know this place during the working day

**Review by lance6716 — APPROVED** [review]

**Review by purelind — APPROVED** [review]
LGTM

**Pull request merged**